### PR TITLE
chore: Clean up user management in tracker tests [TECH-890]

### PR DESCRIPTION
**Pull request by enricocolasante**
This PR is just cleaning up a little bit the tests in tracker module fundamentally doing 2 things:

- Removing `AbstractImportValidationTest` as it was just an indirection for `TrackerTest` injecting admin user in tracker params. Now admin user is explicitly injected in the setup using `injectAdminUser()` method.
- Make some remaining tests extends TrackerTest in order to remove redundant code

### CODE REVIEW

**Review by ameenhere — APPROVED** [review]

File: dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/DhisConvenienceTest.java [hunk 1]
```diff
@@ -2528,6 +2528,11 @@ protected final User createAndInjectAdminUser( String... authorities )
         return user;
     }
 
+    protected void injectAdminUser()
+    {
+        injectSecurityContext( userService.getUser( "M5zQapPyTZI" ) );
+    }
```
Comment (lucaCambi77) [inline dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/DhisConvenienceTest.java:2577]: maybe admin user uid as a static string?

Comment (enricocolasante) [reply dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/DhisConvenienceTest.java]: Done

**Review by teleivo — APPROVED** [review]
Added minor suggestions you can follow up on

**Comment by teleivo** [comment]
@enricocolasante you did not remove AbstractImportValidationTest. Is that one now unused and can be removed?

**Pull request merged**